cli/cli/Commands/Project/Deps/DepsCommand.cs

```diff
@@ -0,0 +1,25 @@
+using Microsoft.Diagnostics.Tracing.Parsers.MicrosoftWindowsTCPIP;
+
+namespace cli.Commands.Project.Deps;
+
+public class DepsCommandArgs : CommandArgs
+{
+	
+}
+
+public class DepsCommand : AppCommand<DepsCommandArgs>, IEmptyResult

```

Comment:
little nitpick, we have a `CommandGroup` type that can be used, so this could be similar to
```csharp

public class ProfilingCommand : CommandGroup, IStandaloneCommand
{
	public override bool IsForInternalUse => true;

	public ProfilingCommand() : base("profile", "Commands for inspecting profiling reports")
	{
	}

}

```

Reply:
although, we don't want the `IsForInternalUse` flag in this case.